webapp/channels/src/components/team_selector_modal/team_selector_modal.tsx

```diff
@@ -287,9 +286,9 @@ export class TeamSelectorModal extends React.PureComponent<Props, State> {
                         componentClass='h1'
                         id='teamSelectorModalLabel'
                     >
-                        <FormattedMarkdownMessage
-                            id='add_teams_to_scheme.title'
-                            defaultMessage='Add Teams to **Team Selection** List'
+                        <FormattedMessage
+                            id='teamSelectorModal.title'
+                            defaultMessage='Add Teams to Team Selection List'
```

Comment:
we should bold the `Team Selection`

Reply:
Yeah I saw that since this was entirely a h1 component all of it was already in bold, making `Team Selection` bold didn't change anything so I thought it redundant, do you still want me to add the `<b>` tag?

Reply:
Oh so if there is not difference visually then its best the redundant.

Reply:
Didn't understand that message, do I add the `<b>` tag?

Reply:
no i mean lets remove the tag